test/test.py

```diff
@@ -92,5 +93,6 @@ def assertRow(row, expected_date, expected_supp, expected_used):
             self.assertEqual(row.supplied, expected_supp)
             self.assertEqual(row.used, expected_used)
 
+        #! NIJZ is changing data tests could fail in the future
         assertRow(data[9], datetime.datetime(2021, 1, 4), 39780, 13248)
-        assertRow(data[22], datetime.datetime(2021, 1, 17), 60870, 48798)
+        assertRow(data[22], datetime.datetime(2021, 1, 17), 60870, 48799)
```

Comment:
See also https://github.com/sledilnik/py-cepimose/issues/2

Reply:
Instead of expecting _exactly_ 48799 we could be happy with anything _close enough_, eg 48798...48820

Reply:
Some tolerance added to tests in #4